src/main/java/com/marcinsz/eventmanagementsystem/json/JsonService.java

```diff
@@ -0,0 +1,108 @@
+package com.marcinsz.eventmanagementsystem.json;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.marcinsz.eventmanagementsystem.dto.EventDto;
+import com.marcinsz.eventmanagementsystem.exception.InvalidJsonFileException;
+import com.marcinsz.eventmanagementsystem.exception.UserNotFoundException;
+import com.marcinsz.eventmanagementsystem.mapper.EventMapper;
+import com.marcinsz.eventmanagementsystem.model.Event;
+import com.marcinsz.eventmanagementsystem.model.User;
+import com.marcinsz.eventmanagementsystem.repository.EventRepository;
+import com.marcinsz.eventmanagementsystem.repository.UserRepository;
+import com.marcinsz.eventmanagementsystem.request.CreateEventRequest;
+import com.marcinsz.eventmanagementsystem.service.JwtService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.validation.annotation.Validated;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+@Service
+@Slf4j
+@Validated
+@RequiredArgsConstructor
+public class JsonService {
```

Comment:
ta nazwa taka ogolne nic nie mowiaca, moze daloby rade jakos inaczej to nazwac? jakie to dane obsluguje?